pkgs/applications/networking/mailreaders/claws-mail/default.nix

```diff
@@ -85,6 +86,9 @@ stdenv.mkDerivation {
     ++ optional (!enablePluginVcalendar) "--disable-vcalendar-plugin"
     ++ optional (!enableSpellcheck) "--disable-enchant";
 
+  wrapPrefixVariables = [ "GIO_EXTRA_MODULES" ];
+  GIO_EXTRA_MODULES = "${glib_networking}/lib/gio/modules";
```

Comment:
This is done automatically by `wrapGAppsHook`. Just add `glib_networking` to `buildInputs`.